Assets/Scripts/Meteor/MeteorMove.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MeteorMove : MonoBehaviour
+{
+    [SerializeField]
+    private float m_Speed;
+    [SerializeField]
+    private GameObject m_ImpactPrefab;
+    [SerializeField]
+    private Transform m_EndPoint;
+    [SerializeField]
+    private Transform m_StartPoint;
+
+    private void Update()
+    {
+        transform.position = Vector3.Lerp(m_StartPoint.position, m_EndPoint.position, 2.5f);
+    }
+
+    private void OnCollisionEnter(Collision collision)
```

Comment:
This should be an `OnTriggerEnter`. Your prefab should also use trigger as its collider. I've spoken to @robytanama about this in the past and he will be able to explain to you in more detail.